app/mailers/proposals_mailer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class ProposalsMailer < ApplicationMailer
+  include Decidim::SanitizeHelper
+  include Decidim::ComponentPathHelper
+
+  layout "decidim/mailer"
+
+  def notify_massive_import(proposals_collection, user)
+    @organization = user.organization
+    @participatory_space_title = decidim_sanitize_translated(proposals_collection.first.participatory_space.title) if proposals_collection.first.present?
```

Comment:
extract `proposals_collection.first` to a variable and avoid doing multiple times.

Reply:
Done like previous suggestion, thanks!